docs/libertyExtensions.md

```diff
@@ -14,7 +14,7 @@ The [deploy](deploy.md), [undeploy](undeploy.md), [libertyPackage](libertyPackag
 | --------- | ---- | ----- | ----------- | ---------|
 | baseDir | String | 3.0 | The base installation directory. The actual installation directory of WebSphere Liberty server will be `${baseDir}/wlp`. The default value is `${project.buildDir}`. This was moved from the properties in the `install` block in version 3.0.| No |
 | cacheDir | String | 3.0 | The directory used for caching downloaded files such as the license or `.jar` files. The default value is `${java.io.tmpdir}/wlp-cache`. This was moved from the properties in the `install` block in version 3.0.| No |
-| installDir | String | 1.0 | Path to the WebSphere Liberty server installation `wlp` directory. To use a pre-installed version of Liberty, set this property to the path of the Liberty `wlp` directory, including `wlp` in the path. | No |
+| installDir | String | 1.0 | Path to the WebSphere Liberty server installation `wlp` directory. To use a pre-installed version of Liberty, set this property to the path of the Liberty `wlp` directory, including `wlp` in the path. <br /> <br /> Additionally, `installDir` can be specified in the `gradle.properties` file or from the command line using `-Pliberty.installDir={wlp}`. | No |
```

Comment:
I don't think I would put `{wlp}` in the example, and let's remove the break.

Reply:
Also, installLiberty.md might be a good place to show an example usage of the installDir property. That is where the `runtime` properties for specifying groupId, artifactId and version for the Liberty runtime are discussed in more detail.